<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HASH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="md5CustomButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5Cus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="md5LibButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5Lib</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sha256Button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-256</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sha3Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>450</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blake2Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BLAKE2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
